### PR TITLE
give it a try with a higher memory limit

### PR DESCRIPTION
**Pull request by ElectricMaxxx**

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
This PR solved the issue, but dos not solve the question for the why. And that change should be done in our dev-kid.

**Comment by ElectricMaxxx** [comment]
@dbu what do you think here? it dies by memory issues while installing lowest. Do we have an issue or should be introduce a memory_limit tweak as i had one commit before?

**Comment by dbu** [comment]
yeah, i guess we should do the unlimited memory thing. the problem will be that there are too many possible variants with all the sonata dependencies.

File: .travis.yml [hunk 1]
```diff
@@ -57,10 +57,8 @@ before_install:
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require symfony/symfony:${SYMFONY_VERSION} --no-update; fi
   - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
 
-install:
-  - phpenv config-add travis.php.ini
-  - php -ini | grep memory_limit
-  - travis_wait composer update --prefer-dist $COMPOSER_FLAGS
+install: composer update --prefer-dist $COMPOSER_FLAGS
```
Comment (dbu) [inline .travis.yml]: i think i messed up on rebasing. but do we still need the unlimited memory?

Comment (ElectricMaxxx) [reply .travis.yml]: you could try it, but you should do it by dev-kit, where i added an option to do those stuff.

Comment (dbu) [reply .travis.yml]: according to the build, we seem to not need it here.

is the thing with the php.ini now in the dev-kit and done to all bundles?

Comment (ElectricMaxxx) [reply .travis.yml]: It is defined in the project config for a single package.

**Pull request closed without merging**